main.js

```diff
@@ -668,7 +668,6 @@ define([
                 this.clearActiveStateForLayerTools(selector);
                 $(el).find('i').addClass('active');
                 $(el).closest('[data-layer-id]').addClass('active');
-                this.rebuildTree();
```

Comment:
Originally, I wanted to keep all UI state in `this.state`, and then re-render the tree any time there is a state change. I'd like to investigate why we seem to be going in another direction (updating UI via jQuery selectors instead of redrawing the whole tree).